libs/saml2-web-sso/src/SAML2/WebSSO/API/UnvalidatedSAMLStatus.hs

```diff
@@ -0,0 +1,39 @@
+-- | This is a partial implementation of Web SSO using the HTTP Post Binding [2/3.5].
+--
+-- The default API offers 3 end-points: one for retrieving the 'AuthnRequest' in a redirect to the
+-- IdP; one for delivering the 'AuthnResponse' that will re-direct to some fixed landing page; and
+-- one for retrieving the SP's metadata.
+--
+-- There are other scenarios, e.g. all resources on the page could be guarded with an authentication
+-- check and redirect the client to the IdP, and make sure that the client lands on the initally
+-- requested resource after successful authentication.  With the building blocks provided by this
+-- module, it should be straight-forward to implemented all of these scenarios.
+--
+-- This module works best if imported qualified.
+--
+-- FUTUREWORK: servant-server is quite heavy.  we should have a cabal flag to exclude it.
```

Comment:
```suggestion
```
Now that we moved the library to `wire-server`, this is probably not important anymore?